samples/aws-codestar-lambda/src/pedestallambda/service.clj

```diff
@@ -0,0 +1,80 @@
+(ns pedestallambda.service
```

Comment:
missing . in namespace

Reply:
This whole sample needs to be rewritten against the new `aws` module.

Reply:
We should do that before cutting a release.

Reply:
Agreed.